pkg/infrastructure/attribute_manager.go

```diff
@@ -76,8 +101,8 @@ func validateStructure(bytes []byte, taxonomySchema string) error {
 
 // GetAttribute returns an infrastructure attribute based on the attribute and instance names
 func (m *AttributeManager) GetAttribute(name taxonomy.Attribute, instance string) *taxonomy.InfrastructureElement {
-	for i, element := range m.Infrastructure.Items {
-		if element.Attribute == name && element.Instance == instance {
+	for i := range m.Infrastructure.Items {
```

Comment:
Why not assign to element in the for loop as you did originally?

Reply:
go-lint complains...

Reply:
@roytman Can we fix golint to not complain?  This is making the code "uglier" and less efficient.

Reply:
will check it

Reply:
I did some refactoring to make it look better

Reply:
Of course, we can silence the linter here, but ....